src/components/Main.jsx

```diff
@@ -1,19 +1,40 @@
-
+import Twitter from "../image/twittericon.png";
+import Facebook from "../image/facebookicon.png";
+import Instagram from "../image/instagramicon.png";
+import Linkedin from "../image/linkedinicon.png";
+import GitHub from "../image/gitHubicon.png";
+import Icon from "../image/emailicon.png";
 
 export default function MainContent() {
-    return(
-        
-        <div className="maincontent">
-           <img className="maincontent__elkay" src="./images.elkay.png" alt="elkay" />
-           <h1 className="maincontent__name">Litany </h1>
-           <h2 className="maincontent__ job-title">Frontend Developer</h2>
-           <h3 className="maincontent__email">litanykato.website</h3>
-           <h2 className="maincontent__title">About</h2>
-           <p className="maincontent__paragraph">I am a frontend developer with a particular interest in making things simple and automating daily tasks. I try to keep up with security and best practices, and am always looking for new things to learn.</p>
-           <h2 className="maincontent__title">Interests</h2>
-           <p className="maincontent__paragraph">Food expert. Music scholar. Reader. Internet fanatic. Bacon buff. Entrepreneur. Travel geek. Pop culture ninja. Coffee fanatic.</p>
+  return (
+    <>
+      <div className="maincontent">
+        <h1 className="maincontent__name">Litany Kato</h1>
+        <h2 className="maincontent__job-title">Frontend Developer</h2>
+        <h3 className="maincontent__email">litanykato.website</h3>
+        <div class="rectangley">
+          <button className="maincontent__button-text"> Email</button>
+          <img className="emailicon" src={Icon} alt="Email Icon" />{" "}
+        </div>
+        <h2 className="maincontent__title">About</h2>
+        <p className="maincontent__paragraph">
+          I am a frontend developer with a particular interest in making things
+          simple and automating daily tasks. I try to keep up with security and
+          best practices, and am always looking for new things to learn.
+        </p>
+        <h2 className="maincontent__title">Interests</h2>
+        <p className="maincontent__paragraph">
+          Food expert. Music scholar. Reader. Internet fanatic. Bacon buff.
+          Entrepreneur. Travel geek. Pop culture ninja. Coffee fanatic.
+        </p>
+        <div class="rectangle">
+          <img className="socialmedia" src={Twitter} alt="twitter-icon" />
```

Comment:
See https://accessibility.huit.harvard.edu/describe-content-images for alt naming, 
```suggestion
          <img className="socialmedia" src={Twitter} alt="Check out our twitter" />
```